added Scanx(dest...) and iter.Scanx(dest...) methods

derkan opened the pull request:
Will give opportunity to scan selected columns without defining records.
```go
        // Select single record values
        authorName := ""
        title := ""
        err = db.SelectFrom("books").
                Where("title = ?", bookTheHobbit.Title).
                Columns("author","title")
                Scanx(&authorName, &title)
        if err == sql.ErrNoRows {
                // sql.ErrNoRows is only returned when the target is a single instance
                fmt.Println("Book not found !")
        } else {
                panicIfErr(err)
        }
```

samonzeweb commented:
Hi Erkan,

I'll see in details the PR later (in a few day), I'm overwhelmed. But I'll do it seriously.

Just two things now :

* the readme code will not compile, at least a colon is missing. Please executes it to be sure all is fine (just replace `samonzeweb`with `derkan` in import paths to use your codebase).
* why `Scanx` and not  simply `Scan` ? (just a question).

I'll also see the NullXXXX parts. But it's better to have separates PR for separates things ;)

Thanks for your contribution.
Just a side question : did you use godb in production ? (I hope the subject isn't confidential).

derkan commented:
Hi Samuel,
- Not using yet, but as you see from PR's I'm preparing to use it. I've tested some other ORM/ODM's but godb is lightest, hackable and has a nice SQL builder syntax. 
- At previous PR I've named it as `Scan` but, you have already used this method name to populate records in iterations. So, to use same name for same operation I renamed that method to `Scanx`.
- `Nullxx` part: I accidentally pushed my local commit, so PR is also updated, reverted it back.
- patched README.md
Thanks.
